core/storage/statistics/gae_models.py

```diff
@@ -1336,6 +1396,16 @@ def create(cls, exp_id, exp_version, unresolved_issues):
         exp_issues_instance.put()
         return instance_id
 
+    @staticmethod
+    def get_model_association_to_user():
+        """Suggested new Takeout docstring for ExplorationIssuesModel: All
```

Comment:
Drop "Suggested new Takeout docstring for ExplorationIssuesModel: "

Reply:
done

Reply:
Not done...

Reply:
not sure why this didn't push, its pushing now

Reply:
done